filter.lua

```diff
@@ -117,15 +102,22 @@ local function filter(outbox)
 
 				local message = format('domain="%s",dport="%d",action="%s",reason="%s"',
 					domain, dport, verdict.action, verdict.reason)
-				outbox:send(message)
+				outbox.telegraf:send(message)
```

Comment:
that's an example.. IMO, it's clearer than encapsulating the sends in a `oubox:send(message, frame)`

Reply:
I did not understand this one.